javascript-typing-tutor/styles.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0 auto;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-size: 24px;
+}
+
+.underline {
+  text-decoration: underline;
```

Comment:
it might be helpful to use a `border-bottom` instead of the `text-decoration` it'll be more visible to the user and show up more consistently.